examples/private_information_retrieval/proto/private_information_retrieval.proto

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.private_information_retrieval;
+
+import "google/protobuf/empty.proto";
+
+message Location {
+  float latitude = 1;
+  float longitude = 2;
+}
+
+message PointOfInterest {
+  string name = 1;
+  Location location = 2;
+}
+
+message Request {
+  Location location = 1;
+}
+
+message Response {
+  PointOfInterest point_of_interest = 2;
+}
+
+service PrivateInformationRetrieval {
+  rpc GetNearestPointOfInterest(Request) returns (Response);
```

Comment:
Please do follow standard naming conventions: 
- https://cloud.google.com/apis/design/naming_convention#method_names
- https://cloud.google.com/apis/design/naming_convention#request_and_response_messages

s/GetNearestPointOfInterest/ListNearestPointsOfInterest/
s/Request/ListNearestPointsOfInterestRequest/
s/Response/ListNearestPointsOfInterestResponse/

(or even omit "nearest" entirely)

I know it seems verbose, but otherwise it seems almost impossible to evolve this API in a reasonable way.

Reply:
Done
But the service currently returns a single POI (but probably will be updated to return a list).